deviceModeInit.js

```diff
@@ -153,32 +159,46 @@ var rudderTracking = (function () {
 
   function checkCartAttributes(cart) {
     const { attributes } = cart;
-    if (attributes?.rudderAnonymousId && attributes?.rudderUpdatedAt) {
-      return true;
+    if (attributes?.rudderAnonymousId) {
+      return false;
     }
-    return false;
+    return true;
   }
 
+  function updateCookies() {
+    const expiryInDays = 14;
```

Comment:
Let's keep the expiry for 2 hours only. After each hour we will update it.

Reply:
Done